build/esbuild-build.ts

```diff
@@ -33,3 +35,17 @@ esbuild
     console.error(err);
     process.exit(1);
   });
+
+function createEnvDefines(variableNames: string[]): Record<string, string> {
+  const defines: Record<string, string> = {};
+  dotenv.config();
```

Comment:
Is this necessary if you're already passing Supabase vars in via esbuild defines?

Reply:
Not sure what you mean. About the dotenv reading the `.env` file? It is needed at build time to populate the Supabase values

Reply:
You're right I didn't realize this was the build file